game-core/src/main/java/games/strategy/engine/framework/GameRunner.java

```diff
@@ -261,26 +257,17 @@ public static void hostGame(final int port, final String playerName, final Strin
   /**
    * Spawns a new process to join a network game.
    */
-  public static void joinGame(final GameDescription description, final Messengers messengers, final Container parent) {
+  public static void joinGame(final GameDescription description, final Messengers messengers) {
     final GameDescription.GameStatus status = description.getStatus();
     if (GameDescription.GameStatus.LAUNCHING == status) {
       return;
     }
 
-    final Version engineVersionOfGameToJoin = new Version(description.getEngineVersion());
```

Comment:
Yes it's dropped.
There's a minimal functional change, but overall it should continue to work similarly.
As noted elsewhere in the diff, previously the version was sent inside GameDescription. However because we assume bots have the same version as the lobby this information seems redundant.
This specific code block displays an error message if it thinks the version is incompatible and stops execution.
However when connecting this check is done on the server side as well, so the user gets and error message in any case. From now on after a connection has been established instead of before.